src/components/AppIcon.css

```diff
@@ -23,13 +21,16 @@
 }
 .Title {
   position: absolute;
-  top: 120px;
+  top: 152px;
+  width: 100%;
   padding-left: 10px;
   padding-right: 10px;
-  font-size: 18px;
-  width: 100%;
-  text-align: center;
   color: #333;
+  font-family: Montserrat;
+  font-size: 17px;
+  font-color: #FFFFFF;
```

Comment:
What do you want to change with font-color? if it is to change the color of the font then color is the one you need and it's already set at line 28.

Reply:
Good idea! It is indeed no longer necessary.